src/features/island/collectibles/Collectible.tsx

```diff
@@ -422,6 +424,37 @@ export const READONLY_COLLECTIBLES: Record<CollectibleName, React.FC<any>> = {
       </div>
     </div>
   ),
+
+  "Scary Mike": () => (
+    <div
+      className="absolute bottom-0"
+      style={{ width: `${PIXEL_SCALE * 22}px`, right: `${PIXEL_SCALE * -3}px` }}
+    >
+      <img
+        src={ITEM_DETAILS["Scary Mike"].image}
+        className="w-full pointer-events-auto"
+      />
```

Comment:
```suggestion
      <img
        src={ITEM_DETAILS["Scary Mike"].image}
        className="w-full"
      />
```

**suggestion:** Remove `pointer-events-auto` here. It results in the `img`  getting dragged on Firefox instead of dragging the scarecrow.

If you make this change would you be able to apply it to the `Basic Scarecrow` as well 🙏